### ISSUE TITLE
Guide is hard to follow

### ISSUE THREAD

**Issue by FatihEbibelot** [jiulongw/swift-unity#11]
I maybe asking too much but it seems complicated. If someone didn't check previous guides to integrate unity scene into native app they will be very confused. I wonder If you can create a guide that explains end2end integration of unity 2017.1.1f1 to xcode9 with every steps included?

**Comment by FatihEbibelot** [comment]
Plus if you just download and try to run the app it is not running the error is;

Syncing code from ...
rsync: link_stat "/Classes/." failed: No such file or directory (2)
rsync error: some files could not be transferred (code 23) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-51/rsync/main.c(996) [sender=2.6.9]
rsync: link_stat "/Libraries/." failed: No such file or directory (2)
rsync error: some files could not be transferred (code 23) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-51/rsync/main.c(996) [sender=2.6.9]
Command /bin/sh failed with exit code 23

**Comment by jiulongw** [comment]
Thanks for your comments!

From the error message it seems the environment variables defined in `Exports.xcconfig` is not picked up correctly by Xcode.  Try restart Xcode to see if that fixed the issue.  To avoid this, always build iOS Unity project before even opening the Swift Xcode project.

The main purpose of this repo is the automated demo project, not the guide itself as there are already good guides out there (linked in the readme file).

**Comment by FatihEbibelot** [comment]
In your demo app unity ios  doesn't have the swift build in it. It just have all the asset files how am I going to build unity ios for demoapp?

**Comment by FatihEbibelot** [comment]
Now I'm getting;

ld: warning: ignoring file DemoApp/Unity/Libraries/libil2cpp.a, missing required architecture x86_64 in file DemoApp/Unity/Libraries/libil2cpp.a (2 slices)ld: warning: ignoring file DemoApp/Unity/Libraries/libiPhone-lib.a, missing required architecture x86_64 in file DemoApp/Unity/Libraries/libiPhone-lib.a (3 slices)

and 200 errors. Similarly I'm getting this architecture error for my current project too(which I 
 build by following the guides). Even though architecture is universal it keeps throwing this error? Any idea how to fix it?

**Comment by jiulongw** [comment]
1. Find some Unity tutorial on how to open Unity project, switch to iOS platform and build it.  (Since you're running into link errors I guess you already figured it out)

2. I don't think you can build x86_64 for Unity project, which means you cannot run it in x86_64 based simulator.  You need to run it in an actual device or run in arm64 simulator.  That's why you see these link errors.

**Comment by FatihEbibelot** [comment]
Hi,

I'm just trying to build the product for generic device not trying to run it. I know how to build for IOS in Unity the problem is not that. The biggest problem is linking right now.

**Comment by jiulongw** [comment]
The library provided by Unity does not have x86_64 architectures.  Why do you expect you can build "generic device" with x86_64 support?

Simply remove x86_64 from your Xcode build target.

```
$ lipo -info Libraries/libiPhone-lib.a 
Architectures in the fat file: Libraries/libiPhone-lib.a are: armv7 arm64 armv7s
$ lipo -info Libraries/libil2cpp.a 
Architectures in the fat file: Libraries/libil2cpp.a are: armv7 arm64
```
